### PR TITLE
Added guide for using DCO GitHub UI

### PR DESCRIPTION
**Pull request by seun-beta**
- Added guide for using DCO GitHub UI. 
- Added a that explains how to write good commit messages.
- Fixed some typos.
- Changed the PR checklist

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,14 +39,40 @@ Bug reports and specific feature requests are best discussed in an issue on the
...
+#### Steps to use the DCO browser plugin
+The  [DCO browser plugin](https://github.com/scottrigby/dco-gh-ui) is a handy tool to automatically sign commits created using GitHub. 
+To enable this plugin: 
+
+- Go to the plugin page on the [chrome web store](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo).
+- Alternatively, you could go to the [firefox addon page](https://addons.mozilla.org/en-US/firefox/addon/scott-rigby/) to add the extension to your browser.
+- Once you add the extension, right click on the extension in the toolbar of your browser and select `Options`. 
+- A dialog box will open up as shown below. Fill in your GitHub name (not the handle) and email-id. 
+
+ ![Screenshot of settings for DCO GitHub UI](https://user-images.githubusercontent.com/31214064/55411911-194c8500-5584-11e9-8b56-c8f94b6fa213.png)
```
Comment (GeorgLink) [inline CONTRIBUTING.md:66]: Maybe remove image?

I'm not a fan of images embedded in a CONTRIBUTING.md file for two reasons:
- the file when seen in an editor will not load pictures
- the image is not hosted on the repo, nor does it belong there

Comment (seun-beta) [reply CONTRIBUTING.md]: Hello @GeorgLink I have removed the pictures. 

Thanks.

**Comment by seun-beta** [comment]
Hello @GeorgLink I have removed the images as requested

**Review by GeorgLink — APPROVED** [review]
Awesome! Thanks @seun-beta

**Pull request merged**